ArtemisKit/Sources/Messages/ViewModels/SendMessageViewModels/SendMessageViewModel.swift

```diff
@@ -97,14 +97,33 @@ final class SendMessageViewModel {
 // MARK: - Actions
 
 extension SendMessageViewModel {
+    @MainActor
     func performOnAppear() {
         switch sendMessageType {
         case let .editMessage(message, _):
             text = message.content ?? ""
         case let .editAnswerMessage(message, _):
             text = message.content ?? ""
         default:
-            break
+            do {
+                let conversations = try AnyRepository.shared.fetch(remoteId: Int(conversation.id))
```

Comment:
## Issue

1. User starts typing in a conversation
2. User navigates to reply in a thread
3. User sends draft
4. User goes back
5. **Draft is still in the text field**